man/tpm2_duplicate.1.md

```diff
@@ -21,9 +21,15 @@ These options control the key importation process:
 
   * **-G**, **\--wrapper-algorithm**=_ALGORITHM_:
 
-    The symmetric algorithm to be used for the inner wrapper. Supports:
+    The symmetric algorithm to be used for the inner wrapper if -U is not used.
+    Supports:
     * aes - AES 128 in CFB mode.
     * null - none
+    The key algorithm associated with the public parent if -U is used.
+
+  * **-G**, **\--key-algorithm**=_ALGORITHM_:
+
+    The key algorithm associated with the public parent if -U is used.
```

Comment:
I wonder if we can phase this option out. Wouldn't we just gate this based on `-i` input? if `-i` is used, inner wrapper mode is aes128cfb (as their is no other choice), and if not set the inner wrapper is null?

For backwards compat we can just NOP this option, perhaps issue a warn deprecated, mark deprecated in man page?

Reply:
We can also use `-o` to determine if inner wrapping is performed, `-i` or `-o` implies inner wrapping.

Reply:
Oh but we still need `-G` for determining what to do with symmetric keys since we can't parse a PEM/DER file and figure it out. They may want an AES or KEYEDHASH key, etc....

I guess disregard this train of thought, but this whole tool could be done nicer on the interface but then we have to bump major vers number